apps/docs/app/features/routes/ressurser/kom-i-gang/Component.tsx

```diff
@@ -0,0 +1,45 @@
+import {
+  Heading,
+  Flex,
+  Image,
+  SimpleGrid,
+  Stack,
+  Text,
+} from "@vygruppen/spor-react";
+
+export default function Component() {
+  return (
+    <SimpleGrid columns={{ base: 1, md: 2 }} spacing={8}>
+      <Flex>
+        <Image
+          src="/images/component-examples/component-example-1.png"
+          alt="Componentexample"
```

Comment:
Husk å gi gode alternative tekster på bilder som her - det hadde om ikke annet hjulpet meg forstå hva dette bilder er av nå når jeg ikke kan se det. Det er også fint for brukere som ikke kan se (eller ikke kan se akkurat nå), som får en slags tekstlig versjon av hva du viser på bildet.